aether/python/redis/task.py

```diff
@@ -96,13 +96,16 @@ def add(
         type: str,
         tenant: str
     ) -> bool:
+        _id = task['id']
         key = '_{type}:{tenant}:{_id}'.format(
```

Comment:
in that case... a simpler line `key = f'_{type}:{tenant}:{_id}'`

Reply:
I moved everything to fStrings, which I do love.